localstack/services/sqs/sqs_listener.py

```diff
@@ -457,6 +461,21 @@ def get_message_attributes_md5(cls, req_data):
 
         return message_attr_hash
 
+    # Fixes tags with empty strings as value
+    def fix_missing_tag_values(self, req_data):
+        keys_matched = []
+        for k, v in req_data.items():
+            match = re.match(r"^Tag\.(\d+)\.Key", k)
+            if match:
+                index = match.group(1)
+                tag_val = "Tag.{}.Value".format(index)
+                if tag_val not in req_data.keys():
+                    keys_matched.append(tag_val)
```

Comment:
nit: I guess we could remove `keys_matched` entirely, and simply assign the default value here?
```
                if tag_val not in req_data.keys():
                    req_data[tag_val] = ""
```